pkg/latticestore/latticestore.go

```diff
@@ -166,13 +169,13 @@ func (ds *LatticeDataStore) AddServiceNetwork(name string, account string, arn s
 	ds.lock.Lock()
 	defer ds.lock.Unlock()
 
-	glog.V(6).Infof("AddServiceNetwork name:[%s], account[%s] arn[%s], id[%s]\n", name, account, arn, id)
+	ds.log.Debugf("AddServiceNetwork name:[%s], account[%s] arn[%s], id[%s]", name, account, arn, id)
 
 	Key := ServiceNetworkKey{Name: name, AccountID: account}
 	_, ok := ds.serviceNetworks[Key]
 
 	if ok {
-		glog.V(6).Infof("UpdateServiceNetwork name:[%s], account[%s] arn[%s] id[%s]\n", name, account, arn, id)
+		ds.log.Debugf("UpdateServiceNetwork name:[%s], account[%s] arn[%s] id[%s]\n", name, account, arn, id)
```

Comment:
nit: I notice these have new lines as well. Are these still required? I remember you mentioning these may not be anymore.